### PR TITLE
Remove `__pycache__` and update `.gitignore`

### PR DESCRIPTION
**Pull request by BD103**
This repo popped up in my feed, figured I would check it out. :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,160 @@
...
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
```
Comment (Coding-Cactus) [inline .gitignore]: Don't want to ignore .spec file. Would like contributors to be able to test with correct publish settings.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,160 @@
...
+*.py,cover
+.hypothesis/
+.pytest_cache/
+cover/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
```
Comment (Coding-Cactus) [inline .gitignore]: Not using django

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,160 @@
...
+*.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+
+# Flask stuff:
```
Comment (Coding-Cactus) [inline .gitignore]: Not using flask

File: .gitignore [hunk 4]
```diff
@@ -1,2 +1,160 @@
...
+instance/
+.webassets-cache
+
+# Scrapy stuff:
```
Comment (Coding-Cactus) [inline .gitignore]: Not using scrapy

File: .gitignore [hunk 5]
```diff
@@ -1,2 +1,160 @@
...
+.scrapy
+
+# Sphinx documentation
```
Comment (Coding-Cactus) [inline .gitignore]: Not using sphinx

File: .gitignore [hunk 6]
```diff
@@ -1,2 +1,160 @@
...
+docs/_build/
+
+# PyBuilder
```
Comment (Coding-Cactus) [inline .gitignore]: Not using pybuilder

File: .gitignore [hunk 7]
```diff
@@ -1,2 +1,160 @@
...
+.pybuilder/
+target/
+
+# Jupyter Notebook
```
Comment (Coding-Cactus) [inline .gitignore]: Not using jupyter notebook

File: .gitignore [hunk 8]
```diff
@@ -1,2 +1,160 @@
...
+.ipynb_checkpoints
+
+# IPython
```
Comment (Coding-Cactus) [inline .gitignore]: Not using Ipython

File: .gitignore [hunk 9]
```diff
@@ -1,2 +1,160 @@
...
+profile_default/
+ipython_config.py
+
+# pyenv
```
Comment (Coding-Cactus) [inline .gitignore]: No need to keep commented out stuff.

File: .gitignore [hunk 10]
```diff
@@ -1,2 +1,160 @@
...
+#   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
+#poetry.lock
+
+# pdm
+#   Similar to Pipfile.lock, it is generally recommended to include pdm.lock in version control.
+#pdm.lock
+#   pdm stores project-wide configurations in .pdm.toml, but it is recommended to not include it
+#   in version control.
+#   https://pdm.fming.dev/#use-with-ide
+.pdm.toml
```
Comment (Coding-Cactus) [inline .gitignore]: Not using pdm

File: .gitignore [hunk 11]
```diff
@@ -1,2 +1,160 @@
...
+
+# PEP 582; used by e.g. github.com/David-OConnor/pyflow and github.com/pdm-project/pdm
+__pypackages__/
+
+# Celery stuff
```
Comment (Coding-Cactus) [inline .gitignore]: Not using celery

File: .gitignore [hunk 12]
```diff
@@ -1,2 +1,160 @@
...
+celerybeat-schedule
+celerybeat.pid
+
+# SageMath parsed files
```
Comment (Coding-Cactus) [inline .gitignore]: Not using sagemath

File: .gitignore [hunk 13]
```diff
@@ -1,2 +1,160 @@
...
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
```
Comment (Coding-Cactus) [inline .gitignore]: Not using any unit tests 😜

File: .gitignore [hunk 14]
```diff
@@ -1,2 +1,160 @@
...
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Spyder project settings
+.spyderproject
+.spyproject
+
+# Rope project settings
```
Comment (Coding-Cactus) [inline .gitignore]: Not using rope project

File: .gitignore [hunk 15]
```diff
@@ -1,2 +1,160 @@
...
+.ropeproject
+
+# mkdocs documentation
```
Comment (Coding-Cactus) [inline .gitignore]: Not using mkdocs

File: .gitignore [hunk 16]
```diff
@@ -1,2 +1,160 @@
...
+/site
+
+# mypy
```
Comment (Coding-Cactus) [inline .gitignore]: Not using mypy

File: .gitignore [hunk 17]
```diff
@@ -1,2 +1,160 @@
...
+.mypy_cache/
+.dmypy.json
+dmypy.json
+
+# Pyre type checker
```
Comment (Coding-Cactus) [inline .gitignore]: Not using pyre

File: .gitignore [hunk 18]
```diff
@@ -1,2 +1,160 @@
...
+.pyre/
+
+# pytype static type analyzer
```
Comment (Coding-Cactus) [inline .gitignore]: Not using pytype

File: .gitignore [hunk 19]
```diff
@@ -1,2 +1,160 @@
...
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+*.py,cover
+.hypothesis/
+.pytest_cache/
+cover/
+
+# Translations
```
Comment (Coding-Cactus) [inline .gitignore]: Not using these

File: .gitignore [hunk 20]
```diff
@@ -1,2 +1,160 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
```
Comment (Coding-Cactus) [inline .gitignore:6]: Only need to ignore build/, dist/

File: .gitignore [hunk 21]
```diff
@@ -1,2 +1,160 @@
...
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
```
Comment (Coding-Cactus) [inline .gitignore]: DEFINITELY don't want to ignore lib/

File: .gitignore [hunk 22]
```diff
@@ -1,2 +1,160 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
```
Comment (Coding-Cactus) [inline .gitignore]: Not using any c extensions

**Review by Coding-Cactus — CHANGES_REQUESTED** [review]
Loads of unnecessary stuff

**Comment by BD103** [comment]
Wow that's a lot of changes requested. I copied the `.gitignore` file from [github/gitignore](https://github.com/github/gitignore). Usually I just keep the defaults, but you seem to not want that. I removed the things you requested.

**Review by Coding-Cactus — APPROVED** [review]

**Pull request merged**